styles/section-1.css

```diff
@@ -0,0 +1,133 @@
+.section-1 {
+  margin: auto;
+  padding: 0;
+  width: 1349px;
+  height: 834px;
+  display: block;
+  align-items: center;
+  justify-content: center;
+}
+
+.title {
+  width: 100%;
+  height: 96px;
+  margin-top: 105px;
+  margin-bottom: 20px;
+  align-self: center;
+  font-size: 30px;
+  font-style: italic;
+  font-family: Georgia, 'Times New Roman', Times, serif;
+}
+
+h1 {
+  width: 600px;
+  height: 96px;
+  font-size: 40px;
+  margin: 0 374.5px;
+}
+
+.img {
+  width: 945px;
+  height: 600px;
+}
+
+.img-section-1 {
+  width: 100%;
+  height: 600px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.section-1-img {
+  width: 100%;
+  height: 532px;
+}
+
+.texts-section-1-img {
+  width: 720px;
+  height: 58px;
+  margin-top: 5px;
+  font-size: 15px;
+  color: hsl(0, 0%, 40%);
+  line-height: 20px;
+}
+
+.footer-section {
+  width: 100%;
+  height: 60px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.footer-section-1 {
+  width: 600px;
+  height: 70px;
+  display: block;
+  margin-top: 50px;
+}
+
+h3 {
+  width: 600px;
+  height: 20px;
+  margin-top: 0;
+  font-size: 15px;
+  font-weight: 700;
+  letter-spacing: 0.3px;
+  color: hsl(0, 0%, 20%);
+}
+
+.icons2 {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.p-icons2 {
+  font-size: 13px;
+  color: hsl(0, 0%, 20%);
+  font-weight: 500;
```

Comment:
## Using the Same Rule Repetitively[1]
- [ ] Create a class once and the use in each HTML tag where you need it.
- [ ] Delete all your `display:flex;` properties in entire CSS file
- [ ] Add your HTML tags

**Example:**

**CSS file**

```
.color-x{
  color: hsl(0, 0%, 20%);
}
```
**Html file**

```
<div class="color-x">
```

<hr>

_[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_